apps/percona/deployers/percona-installer.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Deploying percona
+      shell: kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/master/k8s/demo/percona/percona-openebs-deployment.yaml
+
+    - name: Checking percona is in running state
+      shell: kubectl get po --all-namespaces | grep percona | awk '{print $4}'
+      register: result
+      until: "'Running' in result.stdout"
+      delay: 100
+      retries: 100
```

Comment:
Lets use the inbuilt kubectl capabilities (such as -o custom-columns, go-templates. jsonpaths etc., to get resource details like status etc.,). Also, while it is ok to have high number of retries, reduce the delay b/w them. It generally isn't a good idea to have upper bound for timeouts (retries * delay) over 15 min .. if it takes more time then it is a sub-standard setup w/ network-issues OR there maybe actual issues in attaching/mounting PVs.